variables.tf

```diff
@@ -202,3 +202,9 @@ variable "eni_id" {
   default     = ""
   description = "An ID of a network interface which is used as a default route in private route tables (_e.g._ `eni-9c26a123`)"
 }
+
+variable "map_public_ip_on_launch" {
```

Comment:
@alexjurkiewicz we have a common practice of postfixing boolean values with `_enabled`. Can you make this variable `map_public_ip_on_launch_enabled`?

Reply:
@alexjurkiewicz also make sure to run `make init && make readme` after making this change.